packages/flutter_tools/lib/src/disabled_usage.dart

```diff
@@ -0,0 +1,48 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+
+import 'usage.dart';
+
+class DisabledUsage implements Usage {
+  @override
+  bool get isFirstRun => false;
+
+  @override
+  bool get suppressAnalytics => true;
+
+  @override
+  set suppressAnalytics(bool value) { }
+
+  @override
+  bool get enabled => false;
+
+  @override
+  set enabled(bool value) { }
+
+  @override
+  String get clientId => '00000000-0000-4000-0000-000000000000';
```

Comment:
Not `null`?

Reply:
I can try null.  The MockUsage class uses this value, which makes me wonder if it needs to be non-null for some reason.

Reply:
Null works :confetti_ball: